Popup/Popup.module.less

```diff
@@ -141,33 +182,49 @@
 				.padding-start-end(.extract(@agate-popup-closebutton-title-padding, left)[], .extract(@agate-popup-closebutton-title-padding, right)[]);
 			}
 
-			&:not(.withTitle) {
-				.content {
-					.padding-start-end(.extract(@agate-popup-closebutton-content-padding, left)[], .extract(@agate-popup-closebutton-content-padding, right)[]);
-				}
-			}
-
 			&.centered {
 				.title {
-					padding: @agate-popup-closebutton-centered-title-padding;
+					.padding-start-end(.extract(@agate-popup-closebutton-title-padding, left)[], .extract(@agate-popup-closebutton-title-padding, right)[]);
 				}
 			}
 		}
 
 		.buttons {
+			height: @agate-popup-buttons-height;
 			padding: @agate-popup-buttons-padding;
+			margin-top: @agate-popup-buttons-margin-top;
 			background-color: @agate-popup-buttons-bg-color;
 			border-radius: @agate-popup-buttons-border-radius;
 		}
 
 		.content {
+			min-height: @agate-popup-content-min-height;
+			width: @agate-popup-content-width;
 			padding: @agate-popup-content-padding;
 			margin: @agate-popup-content-margin;
 		}
 
 		.closeButton {
-			.position-start-end(@agate-popup-closebutton-position-left, @agate-popup-closebutton-position-right);
-			margin: @agate-component-spacing;
+			height: @agate-popup-closebutton-height;
+			width: @agate-popup-closebutton-height;
+
+			.position(@agate-popup-closebutton-position-top, @agate-popup-closebutton-position-right, @agate-popup-closebutton-position-bottom, @agate-popup-closebutton-position-left);
+			margin: @agate-popup-closebutton-margin;
+
+			.enact-locale-rtl({
+				.position(@agate-popup-closebutton-position-top, @agate-popup-closebutton-position-left, @agate-popup-closebutton-position-bottom, @agate-popup-closebutton-position-right);
+			});
+
+			.client {
+				color: @agate-popup-closebutton-icon-color;
+			}
+
+			.icon {
+				font-size: @agate-popup-closebutton-icon-font-size;
+				line-height: @agate-popup-closebutton-icon-line-height;
+				height: @agate-popup-closebutton-icon-height;
+				width: @agate-popup-closebutton-icon-height;

```

Comment:
I don't understand why we should customize this. As far as I know, shouldn't it be the same as the default size of a large IconButton? If it's not updated, I think we should update the Button instead of overriding it like this. Have we updated the Button?

Reply:
You are right. I added a condition to render large IconButton for Silicon skin and small IconButton for the other skins. Yes, Button was updated. Thank you!